client/stylesheets/base.less

```diff
@@ -1748,11 +1748,24 @@ a.github-fork {
 			color: @primary-font-color;
 		}
 	}
+	.security-banner.U { color: white; background-color: green }
```

Comment:
maybe we can create variables for these in _variables.less and reference them here.  Then we can reuse them in other places.